docs/FeatureManagement/FeatureFlagv2.0.0.schema.json

```diff
@@ -0,0 +1,354 @@
+                    "type": "string",
+                    "title": "Variant Name",
+                    "description": "The name used to refer to a feature variant.",
+                    "default": "",
+                    "pattern": "^(.*)$"
+                },
+                "configuration_value": {
+                    "$id": "#/properties/variants/items/properties/configuration_value",
+                    "type": ["string", "null", "number", "object", "array", "boolean"],
+                    "title": "Variant Configuration Value",
+                    "description": "The inline value set for this feature variant."
+                },
+                "configuration_reference": {
+                    "$id": "#/properties/variants/items/properties/configuration_reference",
+                    "type": "string",
+                    "title": "Variant Configuration Reference",
+                    "description": "The string used to reference a path in the configuration and set the variant's value equal to the value at that path.",
+                    "default": "",
+                    "pattern": "^[a-zA-Z0-9]+(:[a-zA-Z0-9]+)*$"
+                },
+                "status_override": {
+                    "$id": "#/properties/variants/items/properties/status_override",
+                    "type": "string",
+                    "title": "Variant Status Override",
+                    "description": "Overrides the enabled state of the feature according to this value when this variant is used. Does not override the state if value is None.",
+                    "enum": [
+                        "None",
+                        "Enabled",
+                        "Disabled"
+                    ],
+                    "default": "None"
+                }
+            }
+        }
+      },
+      "allocation": {
+        "$id": "#/properties/allocation",
+        "type": "object",
+        "title": "Variant Allocation",
+        "description": "Determines how variants should be allocated for the feature to various users.",
+        "required": [],
+        "properties": {
+            "default_when_disabled": {
+                "$id": "#/properties/allocation/properties/default_when_disabled",
+                "type": "string",
+                "title": "Variant Allocation Default When Disabled",
+                "description": "Specifies which variant should be used when a variant is requested while the feature is considered disabled.",
+                "default": "",
+                "pattern": "^(.*)$"
+            },
+            "default_when_enabled": {
+                "$id": "#/properties/allocation/properties/default_when_enabled",
+                "type": "string",
+                "title": "Variant Allocation Default When Enabled",
+                "description": "Specifies which variant should be used when a variant is requested while the feature is considered enabled and no other variant was allocated to the user.",
```

Comment:
I'm trying to see if we can avoid the "user" concept here so people don't confuse it with the "user" section below. Does this work?

Specifies which variant should be used when ~a variant is requested while~ the feature is considered enabled and no other allocation rules apply.